### ISSUE TITLE
No audio on one-shot playback through Android emulator

### ISSUE THREAD

**Issue by AngelaVilladiego** [google/flutter.plugins#108]
After importing the package and copy-pasting the one-shot playback code, log messages would indicate that everything is working fine. However even though I have sound turned on for my emulator and I can hear system sounds and button clicks, I cannot hear any audio coming from my emulator.

**Comment by monkeyswarm** [comment]
Does it work on a real device?

**Comment by AngelaVilladiego** [comment]
Yes it does work when debugging on my device using API 30. The Emulator is
running API 29.
